src/components/sendV2/form/form.css

```diff
@@ -0,0 +1,170 @@
+@import '../../app/variablesV2.css';
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: 443px;
+  height: auto;
+  background-color: var(--color-white);
+  box-shadow: 0 1px 5px 0 rgba(30, 25, 77, 0.12);
+  border-radius: 6px;
+}
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  height: 65px;
+  border-bottom: 1px solid var(--color-light-gray);
+  margin: 0;
+  padding: 0 16px;
+
+  & > h1 {
+    font-family: var(--heading-font);
+    font-size: var(--font-size-title);
+    font-weight: var(--font-weight-bold);
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 0.83;
+    letter-spacing: normal;
```

Comment:
can remove `font-style`, `font-stretch` and `letter-spacing` since they are set to normal